pkg/packagekit/updates.jsx

```diff
@@ -369,6 +369,7 @@ function UpdatesList(props) {
 
     return (
         <ListingTable className="available"
+                aria-label={_("Available Updates")}
```

Comment:
For consistency, why does Update History use a caption, but this one uses a separate `<h3>`? I suppose we had our reasons back then, but maybe with the new tables this would work more consistently now?

Reply:
Captions shouldn't be used. WebKit cannot render them properly and there are also some accessibility issues.